newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -123,7 +123,6 @@ export type MarketingPlan = {|
   nameByLocale: MessageByLocale,
   descriptionByLocale: MessageByLocale,
   bulletPointsByLocale: Array<MessageByLocale>,
-  creditsAmount: number,
```

Comment:
It was a mistake?

Reply:
yes, the value was duplicated in two places, it's safer to use the one in limits